CHANGELOG.md

```diff
@@ -8,12 +8,156 @@ This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [0.33.0]
+## [0.34.0]
 
 ### "Stable" API changes
 
 (Won't truly be stable until after a 1.0.0 release)
 
+#### Breaking changes
+
+* The parameter `fraction_number` within `pymedphys.Delivery.from_dicom` and
+  `pymedphys.Delivery.to_dicom` has been changed to `fraction_group_number`.
+* The name of the first argument to `pymedphys.Delivery.from_dicom` has been
+  changed from `dicom_dataset` to `rtplan`. It now also accepts either a
+  `pydicom.Dataset` or a filepath.
+* A range of files utilised by `pymedphys.data_path` and related functions
+  that contained the words "mudensity", "mu-density", or "mu_density" have
+  been replaced with "metersetmap".
+* The exceptions that `pymedphys.Delivery.from_mosaiq` raises when either
+  no entry in Mosaiq is found (`NoMosaiqEntries`) or multiple entries are found
+  (`MultipleMosaiqEntries`) now both inherit from `ValueError` instead of the
+  base `Exception` class.
+
+#### Deprecations
+
+* All instances of `mudensity` have been replaced with `metersetmap`. The
+  `mudensity` API is still currently available, but it will be removed in a
+  future release.
+* `pymedphys.Delivery.from_logfile` has been renamed as
+  `pymedphys.Delivery.from_trf`. The previous name is still temporarily
+  available but it will be removed in a future release.
+
+#### New features
+
+* Created an online PyMedPhys GUI. It is accessible from
+  https://app.pymedphys.com. This is in its early stages and most parts of the
```

Comment:
might want to say the hosted version is intended for demonstration purposes only, and isn't appropriate for "daily use" (better that the facility host locally).
You could put in a query to see if there is interest in a hosted service.

Reply:
Yup, I agree. And I should also add the warning that when using the demo sensitive information should not be uploaded. If the user wants to utilise the tools with sensitive information they should install it on their own machine.